liferay-workspace/modules/blade.modellistener/README.markdown

```diff
@@ -0,0 +1,51 @@
+# Model Listener
+
+## What does this sample do when it's deployed?
+
+The Model Listener sample demonstrates adding a custom model listener to a Liferay Portal's out of the box entity. When deploying this sample with no customizations, a custom model listener will be added to `layouts`, listening for `onBeforeCreate` events. This means that any page creation will trigger this listener, which will execute before the new page being created.
```

Comment:
Thanks, @fafonso! That makes sense. This hasn't been published, so I'll make the update.